src/components/products/bubblegum-v2/Hero.jsx

```
@@ -0,0 +1,280 @@
+
+export function Hero({ page }) {
+  return (
+    <BaseHero page={page} light2Off light3Off>
+      <svg
+        width="380"
+        height="243"
+        viewBox="0 0 380 243"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        className="w-full"
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)